app/forms/provider_interface/offer_wizard.rb

```diff
@@ -343,10 +343,8 @@ def validate_ske_conditions(context)
     end
 
     def validate_combined_ske_length
-      if ske_conditions.sum { |sc| sc.length.to_i } > MAX_SKE_LENGTH
-        errors.add(:base, :ske_length_too_long,
-                   max_total_ske_length: MAX_SKE_LENGTH,
-                   ske_count: ske_conditions.length)
+      if ske_conditions.many? && ske_conditions.none? { |sc| sc.length == '8' }
```

Comment:
imho the fact that `length` is represented as a string is a matter that should be invisible outside `models/ske_condition`.

Reply:
@carlosmartinez I agree, but that's an artefact of this class setting it as a string in the first place.  Really it should be stored as a number.